assets/js/modules/analytics/components/module/ModuleOverviewWidget/SiteStats.js

```diff
@@ -126,8 +126,8 @@ SiteStats.options = {
 	width: '100%',
 	chartArea: {
 		height: '80%',
-		width: '100%',
 		left: 60,
+		right: 20,
```

Comment:
Let's try `25` here (and in similar instances below) based on @aaemnnosttv's comment https://github.com/google/site-kit-wp/issues/4944#issuecomment-1092161143.

Reply:
Sounds good to me, I've pushed this change :+1: